ChemoTracker/components/SymptomTracking/Form.js

```diff
@@ -23,95 +28,63 @@ class FormScreen extends Component {
   render() {
     // const {navigate} = this.props.navi
     return (
-       <LinearGradient colors = {[ '#59D0C2','#066368']} style = {{flex:1}}>
-        <View style={{flex:1}}>
-          <View style={styles.circle}>
-          <Text style={styles.grade}>{this.state.gradeMap[this.state.selectedgrade]} </Text></View>
-          <View >
+      <LinearGradient colors={['#59D0C2', '#066368']} style={styles.felx}>
+        <View style={styles.felx}>
+          <View style={styles.circlesContainer}>
+            <View style={styles.circle1}>
+              <View style={styles.circle2}>
+                <View style={styles.circle3}>
+                  <Text style={styles.grade}>{this.state.gradeMap[this.state.selectedgrade]} </Text>
+                </View>
+              </View>
+            </View>
+          </View>
+          <View>
             <Text style={styles.description}> Do not always feel like eating but still able to have regular meals </Text>
           </View>
           <View style={styles.gradeButtons}>
-            <Button style={[styles.button, {backgroundColor: this.state.buttonbg1}]} textStyle={styles.buttonText}
-              onPressIn={()=>{
-                this.setState({buttonbg2:'transparent',buttonbg3:'transparent',buttonbg4:'transparent',buttonbg1:'#fff', selectedgrade:0});
+            <Button style={[styles.button, { backgroundColor: this.state.buttonbg1 }]} textStyle={[styles.buttonText, { color: this.state.buttontxt1 }]}
+              onPressIn={() => {
+                this.setState({
+                  buttonbg1: '#FFF', buttonbg2: 'transparent', buttonbg3: 'transparent', buttonbg4: 'transparent',
+                  buttontxt1: '#FB5B1B', buttontxt2: '#FFF', buttontxt3: '#FFF', buttontxt4: '#FFF',
+                  selectedgrade: 0
```

Comment:
i'd fix the colours here. If they change, it's way easier just the change the colour code in colors.js